interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,212 @@
+        "--output_path", "normalized_data"],
+        "requirements": "src/first_preprocess/preprocess_requirements.txt",
+        "node_count": 1,
+        "processes_per_node": 1
+    },
+    "step2":
+    {
+        "type": "PythonScriptStep",
+        "name": "prepare files step",
+        "script": "prepare_files.py",
+        "input_datareference_path": "normalized_data/",
+        "input_datareference_name": "normalized_data_conditioned",
+        "input_dataset_name": "normalizeddataconditioned",
+        "source_directory": "src/second_preprocess",
+        "arguments": ["split_train_val",
+        "patch",
+        "--label_file", "label.npy",
+        "--output_dir", "splits/",
+        "--stride=25",
+        "--patch=100.",
+        "--log_config", "configs/logging.conf"],
+        "requirements": "src/second_preprocess/prepare_files_requirements.txt",
+        "node_count": 1,
+        "processes_per_node": 1,
+        "base_image": "pytorch/pytorch"
+    },
+    "step3":
+    {
+        "type": "MpiStep",
+        "name": "train step",
+        "script": "train.py",
+        "input_datareference_path": "normalized_data/",
+        "input_datareference_name": "normalized_data_conditioned",
+        "input_dataset_name": "normalizeddataconditioned",
+        "source_directory": "train/",
+        "arguments": ["--splits", "splits",
+        "--train_data_paths", "normalized_data/file.npy",
+        "--label_paths", "label.npy"],
+        "requirements": "train/requirements.txt",
+        "node_count": 1,
+        "processes_per_node": 1,
+        "base_image": "pytorch/pytorch"
+    }
+}
+```
+  
+If you want to create a train pipeline:
+1) All of your steps are isolated
+    - Your scripts will need to conform to the interface you define in the config file
+        - I.e., if step1 is expected to output X and step 2 is expecting X as an input, your scripts need to reflect that
+    - If one of your steps has pip package dependencies, make sure it's specified in a requirements.txt file
+    - If your script has local dependencies (i.e., is importing from another script) make sure that all dependencies fall underneath the source_directory
+2) You have configured your config file to specify the steps needed (see the section below "Configuring a Pipeline" for guidance)
+
+Note: the following arguments are automatically added to any script steps by AzureML:
+```--input_data``` and ```--output``` (if output is specified in the pipeline_config.json)
+Make sure to add these arguments in your scripts like so:
+```python
+parser.add_argument('--input_data', type=str, help='path to preprocessed data')
+parser.add_argument('--output', type=str, help='output from training')
+```
+```input_data``` is the absolute path to the input_datareference_path on the blob you specified.
+  
+# Configuring a Pipeline
+  
+## Train Pipeline
+Define parameters for the run in a config file. See an example [here](pipeline_config.json)
```

Comment:
Same comment as above - this is a pipeline_config file, not config which refers to AML config

Reply:
Sorry, can you clarify what you mean here? The above comment is about downloading the Azure ML config from the portal

Reply:
discussed - replace "config" with "pipeline configuration" in the description

Reply:
Fixed in commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/985d54b411d4fed07055fec9106ea56a2a5924c6